AUTHORS

```diff
@@ -910,19 +913,21 @@ Jeff Harris <techjeffharris@gmail.com>
 Alexander Makarenko <estliberitas@gmail.com>
 Drew Folta <drew@folta.net>
 Myles Borins <myles.borins@gmail.com>
+Jimb Esser <wasteland@gmail.com>
 Dmitriy Lazarev <wKich@users.noreply.github.com>
 Adam Langley <agl@google.com>
 Kári Tristan Helgason <kthelgason@gmail.com>
+ChALkeR <chalkerx@gmail.com>
```

Comment:
A lot of entries like this one could be mapped to a single entry through `.mailmap`.